include/oneapi/dpl/experimental/kt/single_pass_scan.h

```diff
@@ -304,7 +304,7 @@ struct __lookback_submitter<__data_per_workitem, __workgroup_size, _Type, _FlagT
 
 template <bool _Inclusive, typename _InRange, typename _OutRange, typename _BinaryOp, typename _KernelParam>
 sycl::event
-__single_pass_scan(sycl::queue __queue, _InRange&& __in_rng, _OutRange&& __out_rng, _BinaryOp __binary_op, _KernelParam)
+__single_pass_scan(sycl::queue& __q, _InRange&& __in_rng, _OutRange&& __out_rng, _BinaryOp __binary_op, _KernelParam)
```

Comment:
Probably, does it make sense to keep the variable name `__queue` (as before),
to minimize the changes below within `__single_pass_scan` function?
The same comment in regard to another functions which accept `__queue` name.